sale_product_set/readme/USAGE.rst

```diff
@@ -7,18 +7,11 @@ To use this module, you need to:
     - Sort *set* lines, this order will be the default when added into the
       quotation
 
-.. image:: /sale_product_set/static/description/product_set.png
```

Comment:
@heliaktiv Instead of removing the indication images, it should be better to do new ones if visual changes has occurred. Thanks